### PR TITLE
Replace travis workflow by a github action

### PR DESCRIPTION
**Pull request by mlachkar**

### CODE REVIEW

File: .github/workflows/deploy-website.yml [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+jobs:
+  publish:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v1
+      - uses: olafurpg/setup-scala@v2
+      - name: Publish ${{ github.ref }}
+        run: sbt docs/docusaurusPublishGhpages
+        env:
+          GITHUB_DEPLOY_KEY: ${{ secrets.DOC }}
```
Comment (bjaglin) [inline .github/workflows/deploy-website.yml:15]: did you add this? :+1:

**Review by bjaglin — APPROVED** [review]
Nice, I guess other things can be cleaned up now, see https://github.com/scalacenter/scalafix/pull/1136/commits

File: .github/workflows/deploy-website.yml (hunk 1 above)
Comment (mlachkar) [reply .github/workflows/deploy-website.yml:15]: yes.

**Comment by mlachkar** [comment]
gh actions deploying the website https://github.com/scalacenter/scalafix/runs/1319560461?check_suite_focus=true

**Pull request merged**